inst/WORDLIST

```diff
@@ -26,28 +26,32 @@ Bazett's
 BDS
 Biologics
 BMI
-cdisc
 CDISC
 Changelog
 censorings
 codebase
-Codebase
 CRF
+correctitude
```

Comment:
I don't think this is an English word.  Looking at the [vignettes/unit_test_guidance.Rmd](https://github.com/pharmaverse/admiral/blob/96cd52218cf2f75859581792636d694d4f986edb/vignettes/unit_test_guidance.Rmd), where it is used, I think it should be correctness